onlyOffice/src/test/java/org/exoplatform/platform/qa/ui/onlyOffice/OnlyOfficeTestIT.java

```diff
@@ -0,0 +1,157 @@
+package org.exoplatform.platform.qa.ui.onlyOffice;
+
+import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.Configuration;
+import org.exoplatform.platform.qa.ui.commons.Base;
+import org.exoplatform.platform.qa.ui.core.PLFData;
+import org.exoplatform.platform.qa.ui.ecms.pageobject.SiteExplorerHome;
+import org.exoplatform.platform.qa.ui.onlyOffice.pageobject.OnlyOfficeActivityStream;
+import org.exoplatform.platform.qa.ui.onlyOffice.pageobject.OnlyOfficeDocumentApplication;
+import org.exoplatform.platform.qa.ui.onlyOffice.pageobject.OnlyOfficeEditingPage;
+import org.exoplatform.platform.qa.ui.selenium.platform.*;
+import org.exoplatform.platform.qa.ui.selenium.platform.social.SpaceManagement;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import static com.codeborne.selenide.Selectors.*;
+import static com.codeborne.selenide.Selenide.*;
+import static org.exoplatform.platform.qa.ui.selenium.Utils.getRandomNumber;
+import static org.exoplatform.platform.qa.ui.selenium.locator.HomePageLocator.*;
+import static org.exoplatform.platform.qa.ui.selenium.logger.Logger.info;
+import static org.exoplatform.platform.qa.ui.selenium.testbase.LocatorTestBase.ELEMENT_SKIP_BUTTON;
+import static org.exoplatform.platform.qa.ui.selenium.locator.ActivityStreamLocator.ELEMENT_CLOSE_DOCUMENT_PREVIEW1;
+import static org.exoplatform.platform.qa.ui.selenium.locator.ActivityStreamLocator.*;
+
+
+
+public class OnlyOfficeTestIT extends Base {
+
+  HomePagePlatform               homePagePlatform;
+  NavigationToolbar              navigationToolbar;
+  SiteExplorerHome               siteExplorerHome;
+  ManageLogInOut                 manageLogInOut;
+  ConnectionsManagement          connectionsManagement;
+  ActivityStream                 activityStream;
+  SpaceManagement                spaceManagement;
+  OnlyOfficeDocumentApplication  onlyOfficeDocumentApplication;
+  OnlyOfficeActivityStream       onlyOfficeActivityStream;
+  OnlyOfficeEditingPage          onlyOfficeEditingPage;
+
+
+  @BeforeEach
+  public void setupBeforeMethod() {
+
+    homePagePlatform                =new HomePagePlatform(this);
+    navigationToolbar               =new NavigationToolbar(this);
+    siteExplorerHome                =new SiteExplorerHome(this);
+    manageLogInOut                  =new ManageLogInOut(this);
+    spaceManagement                 =new SpaceManagement(this);
+    activityStream                  =new ActivityStream(this);
+    onlyOfficeDocumentApplication   =new OnlyOfficeDocumentApplication(this);
+    onlyOfficeActivityStream        =new OnlyOfficeActivityStream(this);
+    onlyOfficeEditingPage           =new OnlyOfficeEditingPage(this);
+    manageLogInOut.signInCas("root", "gtn");
+    connectionsManagement           =new ConnectionsManagement(this);
+    if ($(ELEMENT_SKIP_BUTTON).is(Condition.exist)) {
+      $(ELEMENT_SKIP_BUTTON).click();
+    }
+  }
+
+  @Test
+  //EditOnline_BTN_US01
```

Comment:
Please use meaningful description. We don't know what this text refers to. It should describe the test with words.